cfgov/jinja2/v1/_includes/molecules/info-unit-2.html

```diff
@@ -108,8 +108,14 @@
                 {%- else -%}
                     {%- set link_text = link.text if link.text
                         else 'Learn More' -%}
-                    {%- set link = '<a class="m-list_link" href="' ~
-                        link.url ~ '">' ~ link_text ~ '</a>' %}
+                    {%- if link.aria_label -%}
+                        {%- set link = '<a class="m-list_link" href="' ~
+                            link.url ~ '" aria-label="' ~
+                            link.aria_label ~ '">' ~ link_text ~ '</a>' %}
+                    {%- else -%}
+                        {%- set link = '<a class="m-list_link" href="' ~
+                            link.url ~ '">' ~ link_text ~ '</a>' %}
+                    {%- endif %}
```

Comment:
Looks like as of Jinja 2.8 we can [now use block assignments to set variables](https://jinja.palletsprojects.com/en/3.0.x/templates/#block-assignments). Can we do something like this to make it a little less repetitive?

```suggestion
{%- set link -%}
    <a class="m-list_link"
        href="{{link.url}}"
        {%- if link.aria_label -%}
            aria-label="{{link.aria_label}}"
        {%- endif %}>{{link_text}}</a>
{%- endset -%}
```

Seems to work in my quick testing, but I'm not sure if there would be any side effects doing it this way.

Reply:
💯 Great update! Done here https://github.com/cfpb/consumerfinance.gov/pull/6528/commits/bed95b797b35527feb3039ded846db963f807273

Reply:
👍